hackathon_site/event/jinja2/event/login.html

```diff
@@ -2,12 +2,14 @@
 
 {% block form_title %}Login{% endblock %}
 
-{% block form_img %}<img src="{{ static('event/images/landing.svg') }}" alt="Login image"/>{% endblock %}
+{% block nav_logout_login_links %}{% endblock %}
+
+{% block form_img %}<img src="{{ static('event/images/login.svg') }}" alt="Login image"/>{% endblock %}
 
 {% set form_action=url("event:login") %}
 
 {% block form_inputs %}
-<div class="row" style="margin-bottom: 0;">
+<div class="row">
     <div class="input-field col s12" style="margin-bottom: 0;">
         <input
```

Comment:
Eventually we'll want to update this to use the `render_field` macro instead of doing it manually (but that requires tweaking the login forms to add the `invalid` css class, so we can do that later)